src/backends/plonky2/mock_signed.rs

```diff
@@ -100,6 +110,10 @@ impl Pod for MockSignedPod {
     fn into_any(self: Box<Self>) -> Box<dyn Any> {
         self
     }
+
+    fn serialized_proof(&self) -> String {
```

Comment:
maybe this could be named `to_string()`?, so that it follows the [`ToString` trait](https://doc.rust-lang.org/std/string/trait.ToString.html) (as it is already happening in the `self.signature.to_string()`)